packages/schematics/src/collection/cypress-project/index.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  apply,
+  chain,
+  mergeWith,
+  move,
+  noop,
+  Rule,
+  SchematicContext,
+  template,
+  Tree,
+  url
+} from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { join, normalize } from '@angular-devkit/core';
+// app
+import {
+  getProjectConfig,
+  readJsonInTree,
+  updateJsonInTree
+} from '../../utils/ast-utils';
+import { cypressVersion, nxVersion } from '../../lib-versions';
+import { replaceAppNameWithPath } from '../../utils/cli-config-utils';
+import { Schema } from '../application/schema';
+
+export interface CypressProjectSchema extends Schema {
+  appProjectRoot: string;
+  e2eProjectName: string;
+  e2eProjectRoot: string;
+}
+
+function checkArchitectTarget(options: CypressProjectSchema): Rule {
+  return (host: Tree): Rule => {
+    const projectConfig = getProjectConfig(host, options.e2eProjectName);
+    if (!projectConfig.architect.e2e) {
```

Comment:
This makes it easier to switch out protractor for cypress but there's bound to be other things in the project which cannot be converted. For example, the tests are written significantly differently. I think claiming we can easily migrate them from protractor to cypress is a false claim. We do the opposite in `jest` and expect the user to manually delete their existing `test` files.